frontend/src/organisms/types.js

```diff
@@ -0,0 +1,13 @@
+export default {
```

Comment:
tohle nebude organismus, ne? jsou to nějaké typy , nic pro vizualizaci

Reply:
přesunuto do utils